app/assets/stylesheets/views/_account-settings.css.scss

```diff
@@ -245,3 +245,14 @@
 .paypal-last-steps-list {
   margin-bottom: lines(1.5);
 }
+
+.paypal-currency-information {
+  padding-left: 30px;
+  position: relative;
+
+  .ss-info {
```

Comment:
@rap1ds There will be an empty space if font-awesome is in use. I don't see this as a problem?

![screen shot 2016-11-08 at 14 57 47](https://cloud.githubusercontent.com/assets/53923/20099972/c36be936-a5c3-11e6-9314-f288117e9f49.png)